### ISSUE TITLE
Specifying cookbook path does not work

### ISSUE THREAD

**Issue by francis826** [chef/cookstyle#65]
# Version:

$ cookstyle --version
Cookstyle 4.0.0
  * RuboCop 0.62.0

# Environment:

Docker Ruby 2.5

`FROM ruby:2.5`

# Scenario:

Trying to specify a cookbook path with cookstyle doesn't recognize any cookbooks

# Steps to Reproduce:

Verify cookbook is present:

```
$ ls site-cookbooks/
common-node

$ ls site-cookbooks/common-node/
Berksfile  LICENSE  Policyfile.rb  README.md  chefignore  metadata.rb  recipes 
```

Run cookstyle against directory:

```
$ cookstyle site-cookbooks/
Inspecting 0 files

0 files inspected, no offenses detected
```

Specifying a file works fine:
```
$ cookstyle site-cookbooks/common-node/metadata.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
```

# Expected Result:

Specifying cookbook directory/path works

# Actual Result:

No cookbooks are recognized

# Notes

Downgrading to `Cookstyle 3.0.2` and `RuboCop 0.55.0` is recognizing paths.

**Comment by tas50** [comment]
@francis826 can you give this a try with a newer cookstyle release. A lot has changed.

**Comment by francis826** [comment]
This is fixed in `5.8.1`, thanks Tim!
